### ISSUE TITLE
Intestinal inflammation

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6190]
**Preferred term label:**
Intestinal inflammation

**Synonyms**
Inflammation of the intestine

**Definition (free text, please give PubMed ID)**
Inflammation in the intestinal tract.
PMID: 9897960 

**Parent term (use hpo.jax.org/app)**
Abnormal intestine morphology 
-> And "Intestinal inflammation" as the parent term of "Inflammation of the large intestine"

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: # 612244

**Comment by pnrobinson** [comment]
adding as HP_4000055
